layouts/shortcodes/profile-card.html

```diff
@@ -2,7 +2,15 @@
 {{- $img := .Get "img" -}}
 <article class="flex-auto mi4 w-30-ns w-40-m w-100 br3 pa3 pa4-ns ma1 mv3 ba b--black-10">
   <div class="tc">
-    <img src="{{ $img }}" style="object-fit: cover; height: 180px; width: 150px;" class="br-100 w4 dib" title="{{ $name }}">
+    {{ with .Page.Resources.Get $img }}

```

Comment:
This does not seem to work. Building the page from your branch results in no output for this block generated at all. Maybe because the images are in `content/people`? Not sure though

Reply:
![image](https://github.com/securityinmobility/security.science/assets/5504713/cbb8c1b9-47dc-48f1-b190-cbf5d2f00665)